schemas/business/en/qrt_0001.json

```diff
@@ -599,19 +599,17 @@
                             "placeholder": "localunits_address",
                             "transforms": [
                               {
+                                "transform": "concatenate_list",
                                 "arguments": {
-                                  "delimiter": ", ",
                                   "list_to_concatenate": [
                                     {
                                       "source": "supplementary_data",
                                       "identifier": "local-units",
-                                      "selectors": [
-                                        "address"
-                                      ]
+                                      "selectors": ["address"]
                                     }
-                                  ]
-                                },
-                                "transform": "concatenate_list"
+                                  ],
+                                  "delimiter": ","
```

Comment:
This would need a space, or it won't format correctly. `", "`

Reply:
Ok, will get that fixed